app/main.py

```diff
@@ -1 +1,86 @@
-#  write your code here
+class Animal:
+    """
+    Class which contains animals. It stores list of in animals list.
+    """
+    animals = []
+
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: type(bool) = True) -> None:
+        """
+        Create instance of Animal.
+        :param name: animal's name.
+        :param is_hungry: boolean that shows if animal is ready to eat
+        with True default value.
+        :param appetite: integer that shows how much food points.
+        this animal need to eat to be full.
+
+        """
+        self.name = name
+        self.is_hungry = is_hungry
+        self.appetite = appetite
+        Animal.animals.append(self)
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        """
+        Function that feed animal if it's is_hungry equal True.
+        It prints how many food points animal ate, return
+        value of animal's appetite and change is_hungry to False.
+        If animal is_hungry is equal to False, function return 0.
+        :return:
+        """
+        if self.is_hungry is True:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        elif self.is_hungry is False:
+            return 0
+        elif self.is_hungry is None:
+            return 0
+        else:
+            return 0
```

Comment:
The checks for `self.is_hungry is None` and the `else` clause are redundant. Since `is_hungry` is initialized as a boolean, it should only be `True` or `False`. Consider removing these checks.